app/src/main/java/org/simple/clinic/newentry/PatientEntryScreen.kt

```diff
@@ -284,7 +285,15 @@ class PatientEntryScreen(context: Context, attrs: AttributeSet) : RelativeLayout
   }
 
   override fun showColonyOrVillagesList(colonyOrVillageList: List<String>) {
+    ArrayAdapter<String>(context, android.R.layout.simple_list_item_1, colonyOrVillageList).also { adapter ->
+      colonyOrVillageEditText.setAdapter(adapter)
+    }
 
+    colonyOrVillageEditText.threshold = 0
```

Comment:
This doesn't make any difference, by default Android applies a threshold of 1 when the threshold is 0. So we can remove this.

https://developer.android.com/reference/android/widget/AutoCompleteTextView?authuser=1#setThreshold(int)